packages/a11y-base/src/aria-id-reference.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright (c) 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+export type AriaIDReferenceConfig = {
+  newId: string | null;
+  oldId: string | null;
+  fromUser: boolean | null;
+};
+
+/**
+ * Sets a new ID reference for a {@link target} element and an ARIA {@link attr | attribute}.
+ *
+ * @param config.newId
```

Comment:
Generally, we rarely use `@param` in TSDoc but it definitely wouldn't harm to keep these.

Reply:
Yeah, I noticed that, but I figured it could be simpler to explain the function better by describing each parameter separetely.